### PR TITLE
Ali/accordion validation issue

### PR DESCRIPTION
**Pull request by ALIRAZA17**
Removed accordion validation issue

refers: #103

it looks like this now

image

### CODE REVIEW

File: src/components/bytes/Edit/EditByteStepper.tsx [hunk 1]
```diff
@@ -45,7 +45,14 @@ function EditByteStepper({ space, byte, byteErrors, errorColor = '#d32f2f', succ
...
         <div id="accordion-collapse" data-accordion="collapse">
           {byte.steps.map((step, index) => (
-            <Accordion key={step.uuid} isOpen={openAccordionIndex === index} label={`Step ${index + 1}: ${step.name}`} onClick={() => toggleAccordion(index)}>
+            <Accordion
+              key={step.uuid}
+              isOpen={openAccordionIndex === index}
+              label={`Step ${index + 1}: ${step.name}`}
+              onClick={() => toggleAccordion(index)}
+              hasError={Boolean(byteErrors?.steps?.[step.uuid])}
+              errorMessage={byteErrors?.steps?.[step.uuid]?.message}
```
Comment (RobinNagpal) [inline src/components/bytes/Edit/EditByteStepper.tsx:54]: Here you can check in the properties of the step. If any of the properties of the step have an error

File: src/components/bytes/Edit/editByteHelper.ts [hunk 2]
```diff
@@ -129,9 +129,14 @@ export function editByteCommonFunctions(setByte: (value: ((prevState: EditByteTy
     updatedByteErrors.steps = undefined;
     byte.steps.forEach((step: ByteStepInput) => {
       const stepError: StepError = {};
-      if (!step.name || step.name.length > nameLimit) {
+      if (!step.name) {
         stepError.name = true;
       }
+
+      if (step.name.length > nameLimit) {
```
Comment (RobinNagpal) [inline src/components/bytes/Edit/editByteHelper.ts:136]: We can may be revert this code

File: src/types/errors/error.ts [hunk 3]
```diff
@@ -19,6 +19,7 @@ export interface StepError {
   name?: boolean;
   content?: boolean;
   stepItems?: Record<string, QuestionError | UserInputError>;
+  message?: string;
```
Comment (RobinNagpal) [inline src/types/errors/error.ts:22]: This can also be reverted

File: src/components/bytes/Edit/EditByteStepper.tsx (hunk 1 above)
Comment (ALIRAZA17) [reply src/components/bytes/Edit/EditByteStepper.tsx:54]: already doing it using that hasError!!

File: src/components/bytes/Edit/editByteHelper.ts (hunk 2 above)
Comment (ALIRAZA17) [reply src/components/bytes/Edit/editByteHelper.ts:136]: done!!

**Pull request merged**